docker/linux/install-scripts/install-docker.sh

```diff
@@ -33,9 +56,6 @@ dos2unix /usr/local/bin/dind
 chmod +x /usr/local/bin/dockerd-entrypoint.sh
 dos2unix /usr/local/bin/dockerd-entrypoint.sh
 
-export VERSION=19
-curl -sSL https://get.docker.com/ | sh
-
```

Comment:
This runs the Docker installation script. We don't need that any longer because we're installing it via `apt-get install` above.